use-cases/vehicle-routing/src/main/java/org/acme/vehiclerouting/rest/VehicleRoutePlanResource.java

```diff
@@ -100,6 +96,50 @@ public String solve(VehicleRoutePlan problem) {
         return jobId;
     }
 
+    @Operation(summary = "Request recommendations to the RecommendedFit API for a new customer.")
+    @APIResponses(value = {
+            @APIResponse(responseCode = "200",
+                    description = "The list of fits for the given customer.",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON,
+                            schema = @Schema(implementation = List.class)))})
+    @POST
+    @Consumes({MediaType.APPLICATION_JSON})
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("recommendation")
+    public List<RecommendedFit<VehicleRecommendation, HardSoftLongScore>> recommendedFit(RecommendationRequest request) {
+        Customer customer = request.solution().getCustomers().stream().filter(c -> c.getId().equals(request.customerId()))
+                .findFirst().orElseThrow(() -> new IllegalStateException("Customer %s not found".formatted(request.customerId())));
```

Comment:
```suggestion
        Customer customer = request.solution().getCustomers().stream()
            .filter(c -> c.getId().equals(request.customerId()))
            .findFirst()
            .orElseThrow(() -> new IllegalStateException("Customer %s not found"
                .formatted(request.customerId())));
```

Let's separate these long fluent calls into more lines.
Easier to read what's happening.